tsdb/chunkenc/histogram_meta.go

```diff
@@ -284,96 +284,6 @@ type Insert struct {
 	bucketIdx int
 }
 
-// Deprecated: expandSpansForward, use expandIntSpansAndBuckets or
-// expandFloatSpansAndBuckets instead.
-// expandSpansForward is left here for reference.
-// expandSpansForward returns the inserts to expand the bucket spans 'a' so that
-// they match the spans in 'b'. 'b' must cover the same or more buckets than
-// 'a', otherwise the function will return false.
-//
-// Example:
-//
-// Let's say the old buckets look like this:
```

Comment:
Would it be possible to add a comment similar to this one? I found the examples helpful for understanding what is going on with the expansion logic